panels/UserPanel.php

```diff
@@ -83,19 +83,26 @@ public function save()
             ]);
         }
 
-        $attributes = array_keys(get_object_vars($data));
-        if ($data instanceof ActiveRecord) {
-            $attributes = array_keys($data->getAttributes());
+        $identityData = $this->identityData($identity);
 
-            $attributeValues = [];
-            foreach ($attributes as $attribute) {
-                $attributeValues[$attribute] =  $data->getAttribute($attribute);
+        // If the identity is a model, let it specify the attribute labels
+        if ($identity instanceof Model) {
+            $attributes = [];
+
+            foreach ($identityData as $attribute => &$value) {
```

Comment:
Sorry for confusion guys, here we with a [bug-ticket](https://github.com/kalessil/phpinspectionsea/issues/166) to cover this.